### PR TITLE
[WS][OKEx] Update feeds

### PR DESCRIPTION
**Pull request by Herklos**
Requires https://github.com/Drakkar-Software/OctoBot-Trading/pull/609

### CODE REVIEW

File: Trading/Exchange/okex_websocket_feed/okex_websocket.py [hunk 1]
```diff
@@ -22,13 +22,10 @@ class OKExCryptofeedWebsocketConnector(exchanges.CryptofeedWebsocketConnector):
     REQUIRED_ACTIVATED_TENTACLES = []
     EXCHANGE_FEEDS = {
         Feeds.TRADES: cryptofeed_constants.TRADES,
-        Feeds.KLINE: Feeds.UNSUPPORTED.value,
```
Comment (GuillaumeDSM) [inline Trading/Exchange/okex_websocket_feed/okex_websocket.py:25]: we don't need to declare them as unsupported anymore ?

File: Trading/Exchange/okex_websocket_feed/okex_websocket.py [hunk 2]
```diff
@@ -22,13 +22,10 @@ class OKExCryptofeedWebsocketConnector(exchanges.CryptofeedWebsocketConnector):
...
         Feeds.TICKER: cryptofeed_constants.TICKER,
-        Feeds.CANDLE: Feeds.UNSUPPORTED.value,
```
Comment (GuillaumeDSM) [inline Trading/Exchange/okex_websocket_feed/okex_websocket.py:27]: Feeds.CANDLE not `Feeds.UNSUPPORTED.value` is not a problem ? considering it has many static usages

File: Trading/Exchange/okex_websocket_feed/okex_websocket.py (hunk 1 above)
Comment (Herklos) [reply Trading/Exchange/okex_websocket_feed/okex_websocket.py:25]: Yes, with the new integration it's not necessary

File: Trading/Exchange/okex_websocket_feed/okex_websocket.py (hunk 2 above)
Comment (Herklos) [reply Trading/Exchange/okex_websocket_feed/okex_websocket.py:27]: No, I with the new integration it works properly

**Review by GuillaumeDSM — APPROVED** [review]

**Pull request merged**